src/kestrel/session.py

```diff
@@ -421,6 +421,8 @@ def _execute_ast(self, ast):
                 #   - complete data source if omitted by user
                 #   - complete input context
                 check_elements_not_empty(stmt)
+                if "input" in stmt:
```

Comment:
Could we iterate the check over all input vars from [`get_all_input_var_names()`](https://github.com/IBM/kestrel-lang/blob/release/src/kestrel/syntax/parser.py#L18) ?